pandas/tests/groupby/test_timegrouper.py

```diff
@@ -1,6 +1,10 @@
-""" test with the TimeGrouper / grouping with datetimes """
-
-from datetime import datetime
+"""
+test with the TimeGrouper / grouping with datetimes
+"""
+from datetime import (
+    datetime,
+    timedelta as dttd,
```

Comment:
does it work to just import `timedelta`, without an alias?

Reply:
Used timedelta without  the alias.